device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json

```diff
@@ -0,0 +1,192 @@
+{
+    "chassis": {
+        "name": "7800R3A-36DM2-LC",
+        "components": [
+            {
+                "name": "Aboot()"
+            },
+            {
+                "name": "Scd(addr=0000:00:18.7)"
+            },
+            {
+                "name": "Scd(addr=0000:08:00.0)"
+            }
+        ],
+        "fans": [],
+        "fan_drawers": [],
+        "psus": [],
+        "thermals": [
+            {
+                "name": "Cpu temp sensor",
+                "controllable": false
+            },
+            {
+                "name": "Center back",
+                "controllable": false
+            },
+            {
+                "name": "Fap0 core0",
+                "controllable": false
+            },
+            {
+                "name": "Fap0 core1",
+                "controllable": false
+            },
+            {
+                "name": "PCIE",
+                "controllable": false
+            },
+            {
+                "name": "Cpu SBTSI",
+                "controllable": false
+            }
+        ],
+        "sfps": [
+            {
+                "name": "qsfp1"
```

Comment:
The names need to be `osfp` instead of `qsfp`.

Reply:
@Staphylo is this the `osfp` applicable for all LCs?

Reply:
Only this one. Currently on our platform `qsfp-dd` and `osfp` are considered as `osfp` by our driver but it doesn't affect any functionality.